Blizzard.IO.RabbitMQ/Rpc/NetqRabbitRpcConnection.cs

```diff
@@ -0,0 +1,52 @@
+﻿using Blizzard.IO.RabbitMQ.Entities.Rpc;
+using EasyNetQ;
+using EasyNetQ.Producer;
+using System;
+using ISerializer = Blizzard.IO.Core.Rpc.ISerializer;
+
+namespace Blizzard.IO.RabbitMQ.Rpc
+{
+    public class NetqRabbitRpcConnection : INetqRpcRabbitConnection<Func<Type, object>>
+    {
+        public IBus RabbitBus { get; }
+        public RpcMessageType MessageType { get; }
```

Comment:
As a convention I believe that it's good to leave a property's identifier as close as you can to its class type, for simplicity purposes, therefore:
`RabbitBus` -> `Bus`
`MessageType` -> `RpcMessageType`

Reply:
I agree about the bus but I really dont like calling a var or property as the same name of the object (It can really be confusing when looking at the source code without visual studio colors)

Reply:
f for the bus
and you can offer suggestion it is a nice feature of github

Reply:
I can see you point, nevertheless I think that this will be a problem to not do this as a convention. When naming structs like `int` or `bool` we usually name them after their purpose (ex: `int numberOfCalls` `bool finishedSomeJob`), but when classes are taken into account, we have to use some convention in order to define names for properties.
In my opinion each extra different name you give to a property can be even more confusing when configuring the class

Reply:
f